config/samples/sample-app-agent.yml

```diff
@@ -0,0 +1,106 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: quarkus-test-agent
+  name: quarkus-test-agent
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: quarkus-test-agent
+  template:
+    metadata:
+      labels:
+        app: quarkus-test-agent
+    spec:
+      containers:
```

Comment:
We probably should define some resource requests (cpu, memory) here for this container.

Reply:
The agent itself should have almost negligible overhead, so this really just depends on whatever the application is that the agent is attached to. In this case it's actually the same as the `sample_app_quarkus`, ie it's just a Quarkus "Hello World" kind of application that just does nothing but serve a static webpage, so the resource requirements are super low.